Frameworks/Security/GenericPasswordItemHandler.mm

```diff
@@ -589,16 +558,14 @@ - (OSStatus)remove:(NSDictionary*)queryDictionary {
         return errSecParam;
     }
 
-    // TODO: Error handling when projections support it.
-    NSArray* credArray = [_vault retrieveAll];
+    WFC::IVectorView<PasswordCredential> credArray = _vault.RetrieveAll();
 
     // Documentation for remove states that it by default removes all matching rows while documentation
     // for query defaults to a single item. Go figure.
-    std::vector<StrongId<WSCPasswordCredentialWrapper>> credentialsToRemove = FindMatchingCredentials(queryDictionary, credArray, true);
+    std::vector<StrongId<PasswordCredentialWrapper>> credentialsToRemove = FindMatchingCredentials(queryDictionary, credArray, true);
 
     for (const auto& credential : credentialsToRemove) {
-        // TODO: Error handling when projections support it.
```

Comment:
Security is nominally below the Foundation line and shouldn't take another dependency. It can't throw exceptions. :frowning:

Doing so will complicate NSURLSession, NSURLConnection, and NSURLProtocol's ability to do authentication exchanges, as those live in Foundation.